usr/src/lib/brand/lx/zone/lx_boot_zone_ubuntu.ksh

```diff
@@ -33,15 +33,15 @@ safe_dir /etc/systemd
 
 # Populate resolv.conf setup files IFF we have resolvers information.
 resolvers=`zone_attr resolvers`
-if [[ $? == 0 ]]; then
+if [ -n "$resolvers" ]; then
```

Comment:
Please use double brackets.

Reply:
Will fix, I copied the style from lx_boot so that one might need fixing too then.

Reply:
That's just a style difference between OmniOS and Joyent - SmartOS doesn't have the extra stuff in lx_boot.